sql/updates/world/master/2021_99_99_99_world_dk_blood_mastery.sql

```diff
@@ -0,0 +1,17 @@
+-- 89832 Death Strike Enabler
+DELETE FROM `serverside_spell` WHERE `Id` = 89832 AND `DifficultyID` = 0;
+INSERT INTO `serverside_spell` (`Id`, `DifficultyID`, `SpellName`, `Attributes`, `CastingTimeIndex`, `ProcFlags`, `ProcChance`, `DurationIndex`, `RangeIndex`, `SchoolMask`) VALUES
+(89832, 0, 'Spell Strike Enabler', 0x1d0, 1 /* CastingTimeIndex: 0/0/0 */, 0xAAAA8 /*ProcFlags: Damage Taken */, 100 /*Proc Chance*/, 21 /*Duration: -1/0/-1 */, 1 /*Range: Self Only*/, 0x1 /* Physical */);
```

Comment:
should have been `Death Strike Enabler`?

Reply:
yup